### PR TITLE
fixes #1 hollow cirlces and font

### PR DESCRIPTION
**Pull request by alexpsu**
fixes #1, the first issue dealing with inconsistent font and bullet points.

### CODE REVIEW

**Comment by nathanallen** [comment]
There is no issue 11, which issue are you referencing?

**Comment by nathanallen** [comment]
Will not merge.

This pull request introduces a redundant style.

See `app/assets/stylesheets/user-styles.css#L31`

File: app/assets/stylesheets/sidebar_admin.css.scss [hunk 1]
```diff
@@ -34,3 +34,7 @@
     display: inline-block;
   }
 }
+
+h3.sidebar_title {
```
Comment (justincastilla) [inline app/assets/stylesheets/sidebar_admin.css.scss]: Please defend your addition to the sidebar_admin.css.scss file.

File: lib/archives_sidebar/app/views/archives_sidebar/_content.html.erb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <% unless sidebar.archives.blank? %>
   <h3 class="sidebar_title"><%= sidebar.title %></h3>
-  <div class="sidebar_body">
-    <ul id="archives">
+  <div class="sidebar-body">
+    <ul>
```
Comment (justincastilla) [inline lib/archives_sidebar/app/views/archives_sidebar/_content.html.erb]: Please defend your removal of the 'archives' id from the <ul> tag.

File: lib/archives_sidebar/lib/archives_sidebar.rb [hunk 3]
```diff
@@ -30,11 +30,11 @@ def parse_request(_contents, _params)
     article_counts = Content.find_by_sql(["select count(*) as count, #{date_func} from contents where type='Article' and published = ? and published_at < ? group by year,month order by year desc,month desc limit ? ", true, Time.now, count.to_i])
 
     @archives = article_counts.map do |entry|
-      month = (entry.month.to_i%12)+1
+      month = (entry.month.to_i%12)
       year = entry.year.to_i
       {
         name: I18n.l(Date.new(year, month), format: '%B %Y'),
-        month: month - 1,
+        month: month-1,
```
Comment (tgaff) [inline lib/archives_sidebar/lib/archives_sidebar.rb:37]: This syntax violates our ruby style guide.  https://github.com/bbatsov/ruby-style-guide#spaces-operators

I think this will fail if you run `rubocop`

File: lib/archives_sidebar/lib/archives_sidebar.rb [hunk 4]
```diff
@@ -30,11 +30,11 @@ def parse_request(_contents, _params)
     article_counts = Content.find_by_sql(["select count(*) as count, #{date_func} from contents where type='Article' and published = ? and published_at < ? group by year,month order by year desc,month desc limit ? ", true, Time.now, count.to_i])
 
     @archives = article_counts.map do |entry|
-      month = (entry.month.to_i%12)+1
+      month = (entry.month.to_i%12)
```
Comment (nathanallen) [inline lib/archives_sidebar/lib/archives_sidebar.rb:33]: How are these changes relevant to issue #1?
